Makefile

```diff
@@ -130,7 +134,11 @@ fmt-validator:  ## Validate go format
 		echo Your code formatting is according to gofmt standards; \
 	fi
 
-checks-validator: lint fmt-validator validate-swagger ## Run all validation/linting steps
+.PHONY: proto-validator
+proto-validator: proto ## build proto and check if diff found
+	@git diff --quiet -- catalog/rocks/catalog.pb.go
```

Comment:
```suggestion
	git diff --quiet -- catalog/rocks/catalog.pb.go
```
# NOT ASHAMED TO BE RUNNING GIT!
OTOH, error messages from `git` will look strange if the last command run and displayed is `protoc`.